pom.xml

```diff
@@ -169,6 +169,181 @@
                         </execution>
                     </executions>
                 </plugin>
+                <plugin>
+                    <artifactId>maven-source-plugin</artifactId>
+                    <version>3.0.1</version>
+                    <executions>
+                        <execution>
+                            <id>attach-sources</id>
+                            <goals>
+                                <goal>jar</goal>
+                            </goals>
+                        </execution>
+                    </executions>
+                </plugin>
+                <plugin>
+                    <artifactId>maven-javadoc-plugin</artifactId>
+                    <version>3.0.0</version>
+                    <executions>
+                        <execution>
+                            <id>attach-javadocs</id>
+                            <goals>
+                                <goal>jar</goal>
+                            </goals>
+                        </execution>
+                    </executions>
+                </plugin>
+                <plugin>
+                    <artifactId>maven-deploy-plugin</artifactId>
+                    <version>2.8.1</version>
+                    <configuration>
+                        <!--
+                          Note that all child projects that are supposed to be deployed need to
+                          define the property main.basedir themselves. If it's not defined, the
+                          child artifact will not be included for deployment to mvn-repo
+                        -->
+                        <altDeploymentRepository>
+                            internal.repo::default::file://${main.basedir}/target/mvn-repo
+                        </altDeploymentRepository>
+                    </configuration>
+                </plugin>
+                <!--
+                    Plugin to auto-generate an Eclipse update-site. The plugin can be run with:
+
+                        mvn org.reficio:p2-maven-plugin:site -N
+
+                    This generates the update-site to 'target/repository'. Afterwards, one can upload this bundle to
+                     a public location.
+                -->
+                <plugin>
+                    <groupId>org.reficio</groupId>
+                    <artifactId>p2-maven-plugin</artifactId>
+                    <version>1.3.0</version>
+                    <executions>
+                        <execution>
+                            <id>default-cli</id>
+                            <configuration>
+                                <featureDefinitions>
+                                    <feature>
+                                        <id>winery</id>
+                                        <version>2.0.0</version>
```

Comment:
Shouldn't this be `2.0.0-SNAPSHOT`?

Reply:
@miwurster what is your opinion?

Reply:
No, it should not. This configuration relates to Eclipse update site we are able to build. An Eclipse update site works totally different than Maven, e.g., no awareness of SNAPSHOT.